calculation/src/main/java/org/cardanofoundation/rewards/calculation/config/NetworkConfig.java

```diff
@@ -0,0 +1,145 @@
+package org.cardanofoundation.rewards.calculation.config;
+
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class NetworkConfig {
+    private int networkMagic;
+    private BigInteger totalLovelace;
+    private BigInteger poolDepositInLovelace;
+    private long expectedSlotsPerEpoch;
+    private BigInteger mainnetShelleyInitialReserves;
+    private BigInteger mainnetShelleyInitialTreasury;
+    private BigInteger mainnetShelleyInitialUtxo;
+    private int genesisConfigSecurityParameter;
+    private int mainnetShelleyStartEpoch;
```

Comment:
Should we remove `mainnet` prefix from these fields as these fields are also applicable for other networks?